client/src/helpers/validateCookie.js

```diff
@@ -0,0 +1,27 @@
+// const validateCookie = () => {
```

Comment:
Do you need this file anymore ?

Reply:
We will do. It just needs changing.

Reply:
now working on it.

Reply:
you will push on this branch/PR or the next one ?

Reply:
next